src/prompt/views.py

```diff
@@ -47,10 +47,7 @@ def get(self, request, id):
 
     def post(self, request, id):
         """Update prompt by id."""
-        try:
-            data = json.loads(request.body)
-        except json.JSONDecodeError:
-            data = request.data
+        data = json.loads(request.body)
```

Comment:
this is inconsistent with what you passed into CreatePromptController; make sure the code is symmetric so put the `json.loads(request.body)` you passed in directly into the controller as it's own variable `data`